prac_05/word_occurences.py

```diff
@@ -0,0 +1,45 @@
+"""
+CP1404 - Ashton Stewart
+word occurences
+Time taken to write all code: 42:43.38
+Time to add commens: 44:46:23
+"""
+
+#projected time: 45mins to an hour (including commenting and debugging)
+#actual time:
+
+user_input = []
+user_input = str(input("What do you want to say?"))
+input_string = user_input.split()
+
+word_checker = [] #acts as a list of all unique words in the sentence
```

Comment:
Do you think perhaps 2 lists and a dictionary might be too much for this question?